src/components/pages/SignUp/SignUpInput/SignUpInput.module.css

```
@@ -0,0 +1,38 @@
+  gap: 10px;
+  padding: 10px 10px;
+  background-color: #696969;
+}
+
+.field {
+  height: 74px;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  padding: 16px 16px;
+  background: rgba(255, 255, 255, 0.5); /*이거는 var()로 작성 안해도 되는건가?*/
```

Stylelint (17.2.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)